client/cmdlft55xx.c

```diff
@@ -1479,20 +1536,32 @@ int CmdT55xxBruteForce(const char *Cmd) {
 			testpwd = bytes_to_num(keyBlock + 4*c, 4);
 
 			PrintAndLog("Testing %08X", testpwd);
+			
+			// Try each downlink_mode if asked to 
+			// donwlink_mode will = 0 if > 3 or set to 0, so loop from 0 - 3
+			for (dl_mode = downlink_mode; dl_mode <= 3; dl_mode++){
+				if ( !AquireData(T55x7_PAGE0, T55x7_CONFIGURATION_BLOCK, true, testpwd, dl_mode)) {
+					PrintAndLog("Aquireing data from device failed. Quitting");

```

Comment:
Correct spelling: "Acquiring ..."

Reply:
Will do.  that was in the original code, i just put the dl_mode loop around it.

Reply:
Again, not your fault. But while you are on it....